### DISCUSSION TITLE
Problem with wired connection

### DISCUSSION THREAD

**Discussion by TitouanTyack** [USC-ACTLab/crazyswarm#420] [Q&A] [answered]
Hi,

I have some trouble to run the crazyswarm code with a wired connection between Vicon pc (Windows 10) and a nuc (Ubuntu 20.04).
The connection is set like that :
![Screenshot from 2021-07-16 15-24-58](https://user-images.githubusercontent.com/85453067/125956309-3f8760bd-9164-4c76-9af3-f04be34bb53e.png)

The vicon_host_name is set with the IP address of the wired connection as you can see here on the mocap helper launch file (it's the same for the hover swarm launch file also) : 
![Screenshot from 2021-07-16 15-23-48](https://user-images.githubusercontent.com/85453067/125956487-c4d5fd4c-eb5c-410f-a3f8-bae2b2b6de89.png)

I tested the ping between the two computer and it is fine.

The chooser python scripts is working well, and I can control a crazyflie 2.1 with the cfclient and a xbox controller. The firmware of the drones and the radio are updated. I followed your documentation.

You can see here the chooser : 
![Screenshot from 2021-07-16 15-29-06](https://user-images.githubusercontent.com/85453067/125957198-f0aae5a3-e262-4130-86a1-96663889757f.png)

And this is the mocap helper : 
![Screenshot from 2021-07-16 15-27-56](https://user-images.githubusercontent.com/85453067/125957277-75535811-f453-444a-8645-38bd9d9ec480.png)
It is staying like that without warning or error.

Also, the code works perfectly with a wireless connection and a vicon bridge.
The hover python scripts worked on a drone.

Do you have an idea why it is not running with a wireless connection ?

Thanks for your help.

**Comment by whoenig** [comment, marked as answer]
You need to use the IP of the Vicon PC, not the one of your Linux PC. You can look up the IP in Windows using `ipconfig` in the terminal.

**Reply by TitouanTyack** [reply]
Hello,

It was the problem. Thanks for your answer.
It is really strange because the ping is working just in one way (Windows computer -> Ubuntu computer), but the Vicon bridge and also the crazyswarm code can have access to the Vicon data from the Windows computer even if the ping doesn't work in this way.

Thank you for your help
